tests/source/fn-custom-7.rs

```diff
@@ -0,0 +1,19 @@
+// rustfmt-fn_args_layout: Block
+// rustfmt-fn_args_density: Vertical
+// rustfmt-fn_arg_indent: Tabbed
+// rustfmt-fn_brace_style: AlwaysNextLine
```

Comment:
Could you also add tests for `rustfmt-fn_args_density: Vertical`, but without the other options?